build.sbt

```diff
@@ -34,4 +23,40 @@ resolvers += "scalaz-bintray" at "http://dl.bintray.com/scalaz/releases"
 
 javaOptions in Test += "-Dconfig.file=conf/application.test.conf"
 
-playDefaultPort := 9101
+PlayKeys.playDefaultPort := 80
+
+// Allow building both ways on Teamcity
+addCommandAlias("riffRaffArtifact", "riffRaffUpload")
+
+packageName in Universal := name.value
+maintainer := "Guardian Developers <dig.dev.software@theguardian.com>"
+packageSummary := "AMIable"
+packageDescription := """Web app for monitoring the use of AMIs"""
+debianPackageDependencies := Seq("openjdk-8-jre-headless")
+
+javaOptions in Universal ++= Seq(
+  "-Dpidfile.path=/dev/null",
+  s"-Dconfig.file=/etc/${name.value}.conf",
+  "-J-XX:MaxRAMFraction=2",
+  "-J-XX:InitialRAMFraction=2",
```

Comment:
This does mean 1/2 of RAM as opposed to 2/3, but that doesn't seem a big problem.